test/e2e/autoscaling_utils.go

```diff
@@ -142,7 +143,6 @@ func (rc *ResourceConsumer) CleanUp() {
 	rc.stop <- 0
 	expectNoError(DeleteRC(rc.framework.Client, rc.framework.Namespace.Name, rc.name))
 	expectNoError(rc.framework.Client.Services(rc.framework.Namespace.Name).Delete(rc.name))
```

Comment:
namespace for suite is automatically deleted in horizontal_pod_autoscaling.go.
when tests are done there is info "STEP: Destroying namespace "e2e-tests-horizontal-pod-autoscaling-xxxxx" for this suite"